firebase.json

```diff
@@ -17,15 +17,13 @@
     "functions": {
       "port": 5001,
       "host": "127.0.0.1"
-
```

Comment:
avoid any changes in the root files, (except when its the main issue)
So remove any changes in `firebase.json` , `.env.sample` or `package.json`